Create 8694.txt (#8919)
hashicorp/consul: .changelog/8694.txt

```release-note:feature

> agent: moved ui config options to a new `ui_config` stanza in agent configuration and added new options for UI metrics.

Reviewer: ```suggestion
agent: moved ui config options to a new `ui_config` stanza in agent configuration and added new options to display service metrics in the UI.
```

```

> ```release-not:deprecation

Reviewer: ```suggestion
```release-note:deprecation
```

> agent: `ui`, `ui_dir` and `ui_content_path` are now deprecated for use in agent configuration files. Use `ui_config.{enable, dir, content_path}` instead. The command arguments `-ui` `-ui-dir` `-ui-content-path` remain supported.

Reviewer: ```suggestion
agent: `ui`, `ui_dir` and `ui_content_path` are now deprecated for use in agent configuration files. Use `ui_config.{enable, dir, content_path}` instead. The command arguments `-ui`, `-ui-dir`, and `-ui-content-path` remain supported.
```
